pycon/finaid/tests/test_forms.py

```diff
@@ -64,3 +69,75 @@ def test_reviewer_message_form(self):
         message = form.save()
         message = FinancialAidMessage.objects.get(pk=message.pk)
         self.assertTrue(message.visible)
+
+
+class TestReceiptForm(TestCase):
+
+    def setUp(self):
+        super(TestReceiptForm, self).setUp()
+        self.user = User.objects.create_user("Foo")
+        self.application = FinancialAidApplication.objects.create(
+            user=self.user,
+            profession="Foo",
+            experience_level=PYTHON_EXPERIENCE_BEGINNER,
+            what_you_want="money",
+            use_of_python="fun",
+            presenting=1,
+        )
+        self.application.save()
+
+    def test_valid_receipt(self):
+        """Checks form is valid with SimpleUploadeFile extension pdf or png."""
+        # Use StringIO to create files
+        png_file = StringIO.StringIO('portable network graphics file')
+        pdf_file = StringIO.StringIO('portable document format file')
```

Comment:
No need to give these StringIO objects initial contents that we're just going to write over.